src/oom/oomd.c

```diff
@@ -10,8 +10,25 @@
 #include "pretty-print.c"
 #include "signal-util.h"
 
+static OomdResourceMetricMask arg_resource_metric_mask = OOMD_RESOURCE_METRIC_MASK_UNSET;
```

Comment:
Yea I admit it's weird. But I needed a way to distinguish between empty and invalid and unset so that the default value only gets set in the latter case.